src/util/commandUtils.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright (c) 2024 IBM Corporation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import * as Path from "path";
+import * as vscode from "vscode";
+import { isWin,mvnCmd,gradleCmd } from "../liberty/devCommands";
+
+/**
+ * Reused from vscode-maven
+ * https://github.com/microsoft/vscode-maven/blob/main/src/mavenTerminal.ts
+ */
+enum ShellType {
+    CMD = "Command Prompt",
+    POWERSHELL = "PowerShell",
+    GIT_BASH = "Git Bash",
+    WSL = "WSL Bash",
+    OTHERS = "Others"
+}
+/**
+ * Reused from vscode-maven
+ * https://github.com/microsoft/vscode-maven/blob/main/src/mavenTerminal.ts
+ */
+function currentWindowsShell(): ShellType {
+    const currentWindowsShellPath: string = vscode.env.shell;
```

Comment:
What does this return? Is it the default terminal or current terminal? What if there are multiple terminals opened in the vs code? Will it return the current terminal?

Reply:
It will return the current terminal details.

Reply:
Please confirm that we require the default terminal instead of current terminal.

Reply:
As per the existing behavior, even if we open multiple type of terminals in vscode, when we perform actions in liberty dashboard, commands will execute in the default terminal type. So we need to get default terminal details to decide the format of command.

https://github.com/user-attachments/assets/e11d283d-77ca-4e32-8188-30a4bd1654bc

Reply:
Okay. Thanks.

Reply:
Okay. Thanks.